src/simulators/statevector/chunk/host_chunk_container.hpp

```diff
@@ -0,0 +1,482 @@
+/**
+ * This code is part of Qiskit.
+ *
+ * (C) Copyright IBM 2018, 2019, 2020.
+ *
+ * This code is licensed under the Apache License, Version 2.0. You may
+ * obtain a copy of this license in the LICENSE.txt file in the root directory
+ * of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Any modifications or derivative works of this code must retain this
+ * copyright notice, and modified files need to carry a notice indicating
+ * that they have been altered from the originals.
+ */
+
+
+#ifndef _qv_host_chunk_container_hpp_
+#define _qv_host_chunk_container_hpp_
+
+#include "simulators/statevector/chunk/chunk_container.hpp"
+
+
+namespace AER {
+namespace QV {
+
+
+//============================================================================
+// host chunk container class
+//============================================================================
+template <typename data_t>
+class HostChunkContainer : public ChunkContainer<data_t>
+{
+protected:
+  AERHostVector<thrust::complex<data_t>>  data_;     //host vector for chunks + buffers
+  std::vector<thrust::complex<double>*> matrix_;     //pointer to matrix
+  std::vector<uint_t*> params_;                      //pointer to additional parameters
+public:
+  HostChunkContainer()
+  {
+  }
+  ~HostChunkContainer();
+
+  HostChunkContainer(const HostChunkContainer& obj){}
+  HostChunkContainer &operator=(const HostChunkContainer& obj){}
```

Comment:
These don't do anything. Are we using them?

Reply:
I removed them